### DISCUSSION TITLE
Aside out of place

### DISCUSSION THREAD

**Discussion by harrelfe** [quarto-dev/quarto-cli#952] [Q&A]
I'm sorry I can't provide a reproducible example (other than the whole report at https://fharrell.com/post/rflow) because it only seems to happen in a very long report with lots of `.aside`s and `.column-margin`s but I'm getting an `.aside` from early in the report appearing much later and overwriting a `.column-margin` callout (the two appear on top of each other).  I hope that gives enough clues for debugging an apparent `Quarto` problem.

**Comment by dragonstyle** [comment]
No worries, I'm happy to look into it (and nice to see such extensive use of the layout, it will be a great test for Quarto!). Can you give me more info about which aside is appearing out of order / in the incorrect location (e.g. some text I can search for, etc...) ? It isn't immediately jumping out to me but I have a couple of theories to investigate once I find it...

**Reply by harrelfe** [reply]
Thanks for looking into it!  I think this will be a good test of the system.  It have code-generated callouts of various kinds and html tables in column margins among other things.  I've loving the result.

The item in question is defined in a 2nd level section Data Dictionary closer to the top: "Users having the `xless` system command installed ..." which on my Firefox linux browser collides with a word in red that I can't read in the level 1 section Analysis level 2 subsection One Continuous and One Categorical Predictor.  I've attached a screenshot.

**HOWEVER** all is well with Google chrome, Brave, and Vivaldi browsers.

**UPDATE** The red word being overwritten is from a construct I put at the top of the document:

```
[`r hypcomment`]{.aside}
```

Where `hypcomment` is defined as 

```
# Create pop-up for how to use hypothes.is
hypcomment <- '<a href="https://hbiostat.org/comment.html"
  target="popup"
	onclick="window.open(\'https://hbiostat.org/comment.html\', \'popup\',
     \'width=450,height=600\'); return false;">
	<small>Comments</small>
</a>'
```

I'll bet that is something that `Quarto` doesn't like.  Other solutions welcomed.  I was trying to add a popup window.

![z](https://user-images.githubusercontent.com/1150682/169913809-56210bf9-2131-455f-a972-94d6edaa08df.png)

**Reply by dragonstyle** [reply]
Sorry for the delay! I've finally had a chance to start looking into this. I started by simply trying to see if I could reproduce the problem on a local browser (linux or mac) when viewing:

https://www.fharrell.com/post/rflow

Firefox on both platforms for me seems to render that side in the expected location. A couple thoughts:

- Perhaps the content has merely changed and as a side effect perhaps that has fixed the issue
- Perhaps our Firefox versions are different - I'm testing on 97.0 on linux and 101.0 on MacOS. Could you confirm what version of Firefox you're using when you see this issue?

Thanks!
